src/webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -286,11 +290,43 @@ const loadJobs = (specifiedVc) => {
     ],
     'deferRender': true,
     'autoWidth': false,
+    'dom': '<\'row\'<\'col-sm-6\'B><\'col-sm-6\'f>>' +
+      '<\'row\'<\'col-sm-12\'tr>>' +
+      '<\'row\'<\'col-sm-5\'i><\'col-sm-7\'p>>',
+    'buttons': [
+      {
+        text: 'Select all',
```

Comment:
I think the "Select all" button label is confused here, since it actually select all jobs in current page.

Reply:
Renamed as 'Select displayed jobs'